packages/rocket-uploads3-store-event-aws-infrastructure/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@boostercloud/rocket-uploads3-store-event-aws-infrastructure",
+  "version": "0.9.5697",
+  "description": "Booster rocket to upload file in S3 and store event in Event Store when it happens.",
+  "keywords": [
+    "Booster",
+    "plugin",
+    "aws",
+    "s3-trigger-lambda"
+  ],
+  "author": "Booster Cloud",
+  "homepage": "https://booster.cloud",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "dist/index.js",
+  "files": [
+    "dist"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/boostercloud/booster.git"
+  },
+  "dependencies": {
+    "@aws-cdk/aws-dynamodb": "1.67.0",
+    "@aws-cdk/aws-lambda": "1.67.0",
+    "@aws-cdk/aws-lambda-event-sources": "1.67.0",
+    "@boostercloud/framework-types": "^0.9.3",
+    "aws-cdk": "1.67.0",
```

Comment:
is it possible to just import @aws-cdk/core? I think this dependency only needs the Stack object